app/views/layouts/application.html.erb

```diff
@@ -14,8 +14,10 @@
         <div class="nav-link">
           <% if current_user.is_admin? %>
             <%= link_to "User List", users_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'users'}" %>
-            <%= link_to "Initiatives", root_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'initiatives'} " %>
+            <% elsif current_user.team_members.present? %>
```

Comment:
This should probably be outdented to the same level as the `if`.

Reply:
We resolved this issue